control-plane-operator/controllers/hostedcontrolplane/storage/envreplace.go

```diff
@@ -55,6 +56,7 @@ var (
 		"LIVENESS_PROBE_CONTROL_PLANE_IMAGE":              "csi-livenessprobe",
 		"KUBE_RBAC_PROXY_CONTROL_PLANE_IMAGE":             "kube-rbac-proxy",
 		"TOOLS_IMAGE":                                     "tools",
+		"MANILA_DRIVER_CONTROL_PLANE_IMAGE":               "csi-driver-manila",
```

Comment:
Can you group this with the other `*_DRIVER_CONTROL_PLANE_IMAGE` variables?

(Also, for other reviewers, the image is (sadly) still called [`csi-driver-manila`](https://github.com/openshift-eng/ocp-build-data/blob/openshift-4.18/images/csi-driver-manila.yml). We should change that to `openstack-manila-csi-driver` but that's not today's problem).

Reply:
done